core/divesoft.cpp

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-2.0
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "errorhelper.h"
+#include "subsurface-string.h"
+#include "gettext.h"
+#include "dive.h"
+#include "divelist.h"
+#include "divelog.h"
+#include "extradata.h"
+#include "format.h"
+#include "libdivecomputer.h"
+
+int divesoft_import(const std::unique_ptr<std::vector<unsigned char>> &buffer, struct divelog *log)
+{
+	int model = 0;
+	if (strncmp((char *)(buffer->data() + 52), "7026", 4) == 0)
```

Comment:
Please add some comments explaining the magic constants.

I who've reversed engineered this format once don't know what you're checking here.